<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>超市收银台-20172308002-王溯长</string>
  </property>
  <widget class="QPushButton" name="buttonCheckOut">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>500</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>结账</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>570</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClear">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>540</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>721</width>
     <height>601</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>商品编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>商品名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>单价</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>总价</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>301</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>信息提示窗口[模拟小票打印等功能]</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="labelUserLogin">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonOpenUM">
      <property name="text">
       <string>会员管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonOpenPM">
      <property name="text">
       <string>仓储管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonOpenSM">
      <property name="text">
       <string>员工管理</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>470</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAddProduct_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>500</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDelRow">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>500</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDelRow_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>540</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
